lib/openhab/core_ext/ruby/date.rb

```diff
@@ -94,9 +104,14 @@ def compare_with_coercion(other)
   # @return [Array, nil]
   #
   def coerce(other)
+    logger.trace { "Coercing #{self} as a request from #{other.class}" }
     return nil unless other.respond_to?(:to_date)
+
+    logger.trace("1 #{[other.to_date, self]}")
     return [other.to_date, self] if other.method(:to_date).arity.zero?
 
+    logger.trace("2 #{[other.to_date(self), self]}")
```

Comment:
these two seem like in-development-only debug logging, and should be removed

Reply:
yup! thanks for spotting them